packages/business-events/.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Not a blocker for this PR, but we should remove this and replace with common biome config

Reply:
I'll update now as we have plenty of time until this is used :)